proxy/src/main/java/com/wavefront/agent/logsharvesting/FlushProcessor.java

```diff
@@ -2,14 +2,7 @@
 
 import com.beust.jcommander.internal.Lists;
 import com.wavefront.common.MetricsToTimeseries;
-import com.yammer.metrics.core.Counter;
-import com.yammer.metrics.core.Gauge;
-import com.yammer.metrics.core.Histogram;
-import com.yammer.metrics.core.Metered;
-import com.yammer.metrics.core.MetricName;
-import com.yammer.metrics.core.MetricProcessor;
-import com.yammer.metrics.core.Timer;
-import com.yammer.metrics.core.WavefrontHistogram;
+import com.yammer.metrics.core.*;
```

Comment:
Also can you add javadoc for this class? not your code but good to have javadoc for easy readability

Reply:
Added javadoc.